osu.Game/Screens/Play/HUD/ArgonHealthDisplay.cs

```diff
@@ -28,15 +29,15 @@ public partial class ArgonHealthDisplay : HealthDisplay, ISerialisableDrawable
         public bool UsesFixedAnchor { get; set; }
 
         [SettingSource("Bar height")]
-        public BindableFloat BarHeight { get; } = new BindableFloat(20)
+        public BindableFloat BarHeight { get; } = new BindableFloat(30)
         {
             MinValue = 0,
             MaxValue = 64,
             Precision = 1
         };
 
         [SettingSource("Bar length")]
-        public BindableFloat BarLength { get; } = new BindableFloat(0.98f)
+        public BindableFloat BarLength { get; } = new BindableFloat(0.35f)
         {
```

Comment:
I don't think these should be changed as they were sane defaults for a more broad use case.

Reply:
Resolved in https://github.com/ppy/osu/pull/25226/commits/91cf237fc12fcbecc6d1345310717407f559a32e